install/space_checker_utils.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import division
+import re
+import os
+import subprocess
+
+def get_wget_download_size(command):
+    """Get the whole download size.
+
+    :param str command: Wget command.
+
+    :return: The actual download size.
+    :rtype: int
+    """
+    try:
+        _, args = command.split(' ', 1)
+    except IndexError:
+        return 0
+
+    download_size_command = "wget --spider --server-response " + args
+    proc = subprocess.Popen(download_size_command.split(), stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT)
+
+    for line in proc.stdout:
```

Comment:
It would be nicer to use a regex in this case, where you would at once:
1. Find the correct HTTP header
2. Check that the value of the HTTP header is an integer
3. Retrieve that value (using regex groups)